front/app/containers/InitiativesShowPage/InitiativeShowPageTopBar.tsx

```diff
@@ -1,12 +1,19 @@
-import React, { memo, useCallback, MouseEvent } from 'react';
-import clHistory from 'utils/cl-router/history';
+import React, {
```

Comment:
We should either have memo or another memoization mechanism, not both. Possibly not either, in most cases :))